### ISSUE TITLE
Throat Entry Book Uses Prod. Index in IPR

### ISSUE THREAD

**Issue by kwellis** [kwellis/woffl#6]
While reviewing the code, I found that the method throat_entry_book under jetplot inside flow is using the productivity index to generate an oil flowrate instead of the vogel method. Is this really the correct method? Not sure why this is written as PIDX instead of Vogel.

**Comment by scottpessetto** [comment]
Good catch. PI likely overstates oil benefit because it’s a straight line
where vogel should tail off
